src/AssemblerReadGraph.cpp

```diff
@@ -207,32 +235,32 @@ bool Assembler::createLocalReadGraph(
         const uint32_t distance1 = distance0 + 1;
 
         // Loop over edges of the global read graph involving this vertex.
-        for(const uint64_t i: readGraph.connectivity[orientedReadId0.getValue()]) {
+        for (const uint64_t i: readGraph.connectivity[orientedReadId0.getValue()]) {
             SHASTA_ASSERT(i < readGraph.edges.size());
-            const ReadGraphEdge& globalEdge = readGraph.edges[i];
+            const ReadGraphEdge &globalEdge = readGraph.edges[i];
 
-            if(!allowCrossStrandEdges && globalEdge.crossesStrands) {
+            if (!allowCrossStrandEdges && globalEdge.crossesStrands) {
                 continue;
             }
 
             // Get the other oriented read involved in this edge of the read graph.
             const OrientedReadId orientedReadId1 = globalEdge.getOther(orientedReadId0);
 
             // If this read is flagged chimeric and we don't allow chimeric reads, skip.
-            if(!allowChimericReads && readFlags[orientedReadId1.getReadId()].isChimeric) {
+            if (!allowChimericReads && readFlags[orientedReadId1.getReadId()].isChimeric) {
                 continue;
             }
 
             // Get alignment information.
-            const AlignmentData& alignment = alignmentData[globalEdge.alignmentId];
+            const AlignmentData &alignment = alignmentData[globalEdge.alignmentId];
             OrientedReadId alignmentOrientedReadId0(alignment.readIds[0], 0);
             OrientedReadId alignmentOrientedReadId1(alignment.readIds[1], alignment.isSameStrand ? 0 : 1);
             AlignmentInfo alignmentInfo = alignment.info;
-            if(alignmentOrientedReadId0.getReadId() != orientedReadId0.getReadId()) {
+            if (alignmentOrientedReadId0.getReadId() != orientedReadId0.getReadId()) {
```

Comment:
I am not necessarily against a space  preceding parentheses after all `if`, but unrelated formatting changes make PRs harder to review. No need to change them back.

Reply:
I think this is another case of the IDE changing things